bundles/org.openhab.binding.ahawastecollection/README.md

```diff
@@ -62,7 +62,7 @@ wasteCollection.items
 
 ```java
 DateTime collectionDay_generalWaste "Next general waste collection" {channel="ahawastecollection:collectionSchedule:wasteCollectionSchedule:generalWaste"}
-DateTime collectionDay_leightweightPackaging "Next lightweight packaging collection" {channel="ahawastecollection:collectionSchedule:wasteCollectionSchedule:leightweightPackaging"}
+DateTime collectionDay_lightweightPackaging "Next lightweight packaging collection" {channel="ahawastecollection:collectionSchedule:wasteCollectionSchedule:lightweightPackaging"}
```

Comment:
see above, this would be a documentation error because the channel is also misspelled in the binding code